emperor/core.py

```diff
@@ -199,15 +205,24 @@ class Emperor(object):
        2013 Nov 26;2(1):16.
 
     """
-    def __init__(self, ordination, mapping_file, dimensions=5, remote=True,
-                 jackknifed=None, ignore_missing_samples=False):
+    def __init__(self, ordination, mapping_file, feature_mapping_file=None,
+                 dimensions=5, remote=True, jackknifed=None,
+                 ignore_missing_samples=False):
 
         self.ordination = ordination
         self.jackknifed = jackknifed
 
         self.mf = mapping_file.copy()
 
-        self._validate_metadata(ignore_missing_samples)
+        self.mf = self._validate_metadata(self.mf, self.ordination.samples,
+                                          ignore_missing_samples)
+
+        # if biplots are to be visualized
+        if self.ordination.features is not None:
+            self.feature_mf = self._validate_metadata(feature_mapping_file,
+                                                      self.ordination.features,
+                                                      False)
```

Comment:
would it be possible to associate the kwarg with the `False` for context?

Reply:
For sure, added.